eng/pipelines/common/templates/pipeline-with-resources.yml

```diff
@@ -79,9 +79,9 @@ extends:
         env:
           ROOTFS_DIR: /crossrootfs/x64
 
-      # We use a CentOS Stream 8 image here to test building from source on CentOS Stream 8.
+      # We use a CentOS Stream 8 image here to test building from source on CentOS Stream 9.
       SourceBuild_centos_x64:
-        image: mcr.microsoft.com/dotnet-buildtools/prereqs:centos-stream8
+        image: mcr.microsoft.com/dotnet-buildtools/prereqs:centos-stream9
```

Comment:
@dotnet/runtime-infrastructure someone please remind me - Do the eng/pipelines/common files come from arcade? If yes, then we cannot modify this file here directly, it needs to be modified in arcade. I just need confirmation.

Reply:
eng/pipelines/common are our files, not arcade's.

Reply:
This also confused me initially.